File: Easty-LeetCodePrblms/PalindromeNumber.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    let num = x;
    
    if(x < 0){
      return false
    }
   
  let sum = 0;
  while(num){
    sum * = 10;
    sum + = num%10;
    num = Math.floor(num/10);
  }
  return x===sum;
};

File: Easty-LeetCodePrblms/PalindromeNumber.java
class Solution {
    public boolean isPalindrome(int x) {
      int num, sum = 0, temp = x;
      
      while(x>0){
        num = x%10;
        sum = (sum*10)+num;
        x = x/10;
      }
      if(temp == sum){
        return true;
      }else{
        return false;
      }
   }
